src/consensus/aft/test/main.cpp

```diff
@@ -18,14 +18,18 @@ using ms = std::chrono::milliseconds;
 using TRaft =
   aft::Aft<aft::LedgerStubProxy, aft::ChannelStubProxy, aft::StubSnapshotter>;
 using Store = aft::LoggingStubStore;
-using StoreSig = aft::LoggingStubStoreSig;
 using Adaptor = aft::Adaptor<Store>;
 
 threading::ThreadMessaging threading::ThreadMessaging::thread_messaging;
 std::atomic<uint16_t> threading::ThreadMessaging::thread_count = 1;
 
 std::vector<uint8_t> cert;
 
+aft::ChannelStubProxy* channel_stub_proxy(const TRaft& r)
+{
+  return (aft::ChannelStubProxy*)r.channels.get();
+}
```

Comment:
Replacing the `((aft::ChannelStubProxy*)r0.channels.get())` pattern with `channel_stub_proxy(r0)`, and in most places giving that a name like `r0c` for readability.